CHANGELOG.md

```diff
@@ -634,6 +652,22 @@ Other minor additions
   record IndexedPoset    {i} (I : Set i) c ℓ₁ ℓ₂ : Set (suc (i ⊔ c ⊔ ℓ₁ ⊔ ℓ₂))
   ```
 
+* Added new functions and records to `Relation.Binary.Indexed`:
+  ```agda
+  record IsIndexedPreorder  (_≈_ : Rel A ℓ₁) (_∼_ : Rel A ℓ₂) : Set (i ⊔ a ⊔ ℓ₁ ⊔ ℓ₂)
+  record IndexedPreorder {i} (I : Set i) c ℓ₁ ℓ₂ : Set (suc (i ⊔ c ⊔ ℓ₁ ⊔ ℓ₂))
+
+  trivialIsIndexedEquivalence : IsEquivalence _≈_ → IsIndexedEquivalence (λ (_ : I) → A) _≈_
```

Comment:
Consider a `Trivial` sub-module rather than this naming convention?

Reply:
I'd actually already done this but forgot to update the changelog :smile: . It's now fixed.